azure-pipelines.yml

```diff
@@ -0,0 +1,12 @@
+trigger:
+  tags:
+    include:
+    - v*.*.*
+
+steps:
+  - task: Npm@1
+    inputs:
+      command: 'publish'
+      workingDir: '.'
+      publishEndpoint: 'NPM Registry'
+      verbose: true
```

Comment:
this won't work as per this issue https://github.com/microsoft/azure-pipelines-tasks/issues/14165
However we implemented a workaround here https://github.com/pagopa/io-functions-commons/pull/131

I suggest you to do it in another PR in which you can refactor the whole pipeline